docs/prod-fluo-setup.md

```diff
@@ -31,35 +31,49 @@ in `modules/distribution/target`:
 Install Fluo
 ------------
 
-When you have a distribution tarball built for your environment, follow these steps
-to install Fluo.
+After you obtain a Fluo distribution tarball, follow these steps to install Fluo.
 
-First, choose a directory with plenty of space and untar the distribution:
+1. Choose a directory with plenty of space and untar the distribution:
 
-    tar -xvzf fluo-1.0.0-beta-2-bin.tar.gz
+        tar -xvzf fluo-1.0.0-beta-2-bin.tar.gz
 
-Verify that your distribution has the same versions of Hadoop & Accumulo as your environment:
-
-    cd fluo-1.0.0-beta-2
-    ls lib/hadoop-* lib/accumulo-*
-
-Next, copy the example configuration to the base of your configuration directory to create
+2. Copy the example configuration to the base of your configuration directory to create
 the default configuration for your Fluo install:
 
-    cp conf/examples/* conf/
+        cp conf/examples/* conf/
+
+    The default configuration will be used as the base configuration for each new application.
 
-The default configuration will be used as the base configuration for each new application.
-Therefore, you should modify [fluo.properties] for your environment.  However, you should not
-configure any application settings (like observers). 
+3. Modify [fluo.properties] for your environment.  However, you should not configure any 
+application settings (like observers). 
 
-NOTE - All properties that have a default are set with it.  Uncomment a property if you want 
+    NOTE - All properties that have a default are set with it.  Uncomment a property if you want 
 to use a value different than the default.  Properties that are unset and uncommented must be
 set by the user.
 
-To run, Fluo requires the Hadoop, Accumulo, and Zookeeper jars for the versions
-running on the cluster where Fluo will run.  In order to configure this a
-classpath must be constructed with these jars for Fluo.  This can be done by
-editing `conf/fluo-env.sh`, which also contains further documentation.  
+4. Fluo needs to know where you have installed Hadoop, Accumulo, and Zookeeper. You can either
```

Comment:
I would reword this a bit.  Fluo needs the versions of Accumulo, Hadoop, and Zookeeper you are running on its classpath.  Knowing they are installed is one way to accomplish this goal.